spec/spec_helper.rb

```diff
@@ -1,7 +1,13 @@
 require 'simplecov'
+require 'coveralls'
+Coveralls.wear!
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
 SimpleCov.start do
-  add_filter 'vendor'
-  add_filter 'spec'
+  add_filter '/spec'
+  add_filter '/vendor'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.